content/en/tutorials/lotus/grafana-dashboard.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Grafana dashboard"
+description: "Lotus supports exporting a wide range of metrics, enabling users to gain insights into its behavior and effectively analyze performance issues. These metrics can be conveniently utilized with aggregation and visualization tools for in-depth analysis. In this tutorial, we show how you can set up Prometheus and Grafana for monitoring and visualizing these metrics"
+lead: "Lotus supports exporting a wide range of metrics, enabling users to gain insights into its behavior and effectively analyze performance issues. These metrics can be conveniently utilized with aggregation and visualization tools for in-depth analysis. In this tutorial, we show how you can set up Prometheus and Grafana for monitoring and visualizing these metrics"
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-lotus"
+weight: 150
+toc: true
+---
+
+- **Prometheus**: Prometheus is an open-source monitoring and alerting toolkit designed for collecting and storing time-series data from various systems and applications. It provides a robust querying language (PromQL) and a web-based interface for analyzing and visualizing metrics.
+
+- **Grafana**: Grafana is an open-source platform for creating, sharing, and visualizing interactive dashboards and graphs. It integrates with various data sources, including Prometheus, to help users create meaningful visual representations of their data and set up alerting based on specific conditions.
+
+## Prerequisites
+
+- You have a Linux or Mac based system.
+- You have root access to install software
+- You have lotus node already running
+
+**Note:** These instructions have been tested on Ubuntu 23.04 and on Mac M1.
+
+## Install and start Prometheus
+
+### On Ubuntu:
+
+```
+# install prometheus
+sudo apt-get install prometheus
+
+# copy the prometheus.yml config to the correct directory
+sudo cp metrics/prometheus.yml /etc/prometheus/prometheus.yml
+
+# start prometheus
+sudo systemctl start prometheus
+
+# enable prometheus on boot (optional)
+sudo systemctl enable prometheus
+```
+
+### On Mac:
+
+```
+# install prometheus
+brew install prometheus
+
+# start prometheus
+prometheus --config.file=lotus/metrics/prometheus.yml
```

Comment:
Where does `lotus/metrics/prometheus.yml` live?  In the Lotus repo right?  How come we don't need to do this on the Linux side?  Maybe use a variable to represent the lotus directory so it's clear that everyone needs to set it for themselves?